src/Routes/web.php

```diff
@@ -12,5 +12,6 @@
     ->group(function () {
         Route::get('/', [InertiaController::class, 'home']);
         Route::get('/register', [InertiaController::class, 'register']);
+        Route::get('/landing', [InertiaController::class, 'view']);
```

Comment:
hmm, why don't we just use the `/` URL for this page?

Reply:
I thought about this, but `login `and `register ` has used `/landing` already, I'll go back to this as a routing hotfix if that's okay. What do you guys think abt this? @Zsaschz @nofamex